plugins/woocommerce/src/Admin/ReportCSVExporter.php

```diff
@@ -236,7 +247,23 @@ public function get_total_rows() {
 	 * Prepare data for export.
 	 */
 	public function prepare_data_to_export() {
-		$request  = new \WP_REST_Request( 'GET', "/wc-analytics/reports/{$this->report_type}" );
+		/**
+		 * Used to add/overwrite report data endpoint.
+		 *
+		 * @since x.x.x
+		 *
+		 * @param string $endpoint The report's data endpoint.
+		 * @param string $type     The report's type.
+		 *
+		 * @returns string The report's endpoint.
+		 */
+		$report_endpoint = apply_filters(
+			'woocommerce_analytics_report_data_endpoint',
+			"/wc-analytics/reports/{$this->report_type}",
+			$this->report_type
+		);
```

Comment:
```suggestion
		$report_endpoint = apply_filters(
			'woocommerce_export_report_data_endpoint',
			"/wc-analytics/reports/{$this->report_type}",
			$this->report_type
		);
```

To match other filters in the file.